services/app-web/src/constants/routes.ts

```diff
@@ -78,12 +83,13 @@ const RoutesRecord: Record<RouteT, string> = {
     SUBMISSIONS_SUMMARY: '/submissions/:id',
     SUBMISSIONS_MCCRSID: '/submissions/:id/mccrs-record-number',
     SUBMISSIONS_REVISION: '/submissions/:id/revisions/:revisionVersion',
-    SUBMISSIONS_QUESTIONS_AND_ANSWERS: '/submissions/:id/question-and-answers',
+    SUBMISSIONS_CONTRACT_QUESTIONS_AND_ANSWERS: 'submissions/:id/question-and-answers',
```

Comment:
```suggestion
    SUBMISSIONS_CONTRACT_QUESTIONS_AND_ANSWERS: '/submissions/:id/question-and-answers',
```

I think we need the `/` at the beginning. Looks like I also missed that for `SUBMISSIONS_RATE_QUESTIONS_AND_ANSWERS`.

Reply:
Made this change to keep consistent